libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackAdapter.cs

```diff
@@ -293,7 +200,7 @@ public async Task ProcessAsync(HttpRequest request, HttpResponse response, IBot
                 return;
             }
 
-            if (!SlackHelper.VerifySignature(_options.ClientSigningSecret, request, body))
+            if (!_slackClient.VerifySignature(_slackClient.Options.ClientSigningSecret, request, body))
```

Comment:
The ClientSigningSecret is already in _slackClient, there's no need to pass it here.

Reply:
my bad